spec.html

```diff
@@ -437,7 +453,7 @@ <h1>Tree Grammar</h1>
 interface ImportSpecifier : Node {
   // The `IdentifierName` in the production `ImportSpecifier :: IdentifierName as ImportedBinding`;
   // absent if this specifier represents the production `ImportSpecifier :: ImportedBinding`.
-  attribute IdentifierName? name;
+  attribute IdentifierDeclaration? name;
```

Comment:
I don't think this is declaration.
it's `X` part of `import { X as Y } from "foo";`, and that's the name of exported thing from the module.
`binding` attribute is the declaration for the current module.

this should be either reference or property name
(depending on how we treat the exported names of modules)

Reply:
Ah, right. I *think* it's a `PropertyName`, then. It's not really bound or scoped, right?